pkg/utils/file.go

```diff
@@ -277,10 +285,22 @@
 			return "", "", err
 		}
 		if _, err := io.Copy(dstFile, srcFile); err != nil {
+			srcFile.Close()  // Close the source file in case of an error
+    		dstFile.Close()  // Close the destination file in case of an error
```

Comment:
Hi @VibhorGits, these `.Close()` calls need to be handled as well.
I advise you to use `defer` function to avoid excessive error handling blocks, like this
```
defer func() {
	// try to close the file; if an error occurs, log the error
	if err := srcFile.Close(); err != nil {
		log.Printf("%v", err.Error())
	}
}()
```